File: week1/임주환/0711/A002.py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:

        list_list = []
        
        for n in range(numRows):
            int_list = []
            for r in range(n + 1):
                nCr = int(math.factorial(n) / (math.factorial(r) * math.factorial(n - r)))

                int_list.append(nCr)
            
            list_list.append(int_list)

        return list_list

File: week1/이상현/7.14/A002.cpp
class Solution {
public:
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>> value;

        for(int i=0; i< numRows; ++i){
            value.push_back(vector<int>(i+1, 1));
        } 
        for(int j=2; j< numRows; ++j)
            for(int k=1; k< value[j].size()-1; ++k)
                value[j][k] = value[j-1][k-1] + value[j-1][k]; 
            
        
        return value; 
    }
};